src/components/CardVideo/CardVideo.jsx

```diff
@@ -0,0 +1,87 @@
+import "../../css/main.css";
+import "./CardVideo.css";
+import { useState } from "react";
+
+export const CardVideo = ({ item }) => {
+  const [isPlay, setIsPlay] = useState(false);
+  const [isMoreOptions, setIsMoreOptions] = useState(false);
+  console.log(item);
+  const { title, thumbnail, channel, profile, views, playbackTime } = item;
+  return (
+    <div className="container-card flex flex-column flex-gap-0-5">
+      <div
+        className="container-img-thumbnail-card cursor-pointer"
+        onMouseLeave={() => setIsPlay(false)}
+        onMouseEnter={() => setIsPlay(true)}
+      >
+        <img
+          className="img-thumbnail-card"
+          src={thumbnail.url}
+          loading="lazy"
+          alt={thumbnail.altTxt}
+        />
+        <div className="duration-card">{playbackTime} min</div>
+
+        <div
+          className={`${
+            isPlay
+              ? "container-overlay-img-thumbnail-card flex flex-justify-center flex-align-center cursor-pointer"
+              : "display-none"
+          }`}
+          onMouseLeave={() => setIsPlay(false)}
+          onMouseEnter={() => setIsPlay(true)}
+        >
+          <button className="btn-transparent">
+            <span className="material-icons-round text-3xl">play_circle</span>
+          </button>
+        </div>
+      </div>
+
+      <div className="flex flex-gap-0-5 text-card-video">
+        <img
+          className="img-profile-card"
+          src={profile.url}
+          loading="lazy"
+          alt={profile.altTxt}
+        />
+        <div className="flex flex-column flex-gap-0-5 flex-grow-1">
+          <p className="text-sm pdt-0-5">{title}</p>
+          <div className="flex flex-align-center flex-gap-0-5">
+            <div className="channel-card-video flex flex-justify-start text-sm">
+              {channel}
+            </div>
+            <span className="text-xs">{views} views</span>
+          </div>
+        </div>
+        <button
+          className={`${
+            isMoreOptions ? "btn-more-vert-card-active" : " "
+          } btn-more-vert-card btn-transparent flex flex-align-center flex-justify-center`}
+          onClick={() => setIsMoreOptions(!isMoreOptions)}
```

Comment:
improvement : just close the previous card option when user click on another card option

Reply:
Done. Thanks for pointing this.